internal/service/fsx/ontap_volume_test.go

```diff
@@ -44,6 +44,7 @@ func TestAccFSxOntapVolume_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "security_style", ""),
 					resource.TestCheckResourceAttr(resourceName, "size_in_megabytes", "1024"),
 					resource.TestCheckResourceAttr(resourceName, "skip_final_backup", "false"),
+					resource.TestCheckResourceAttr(resourceName, "snapshot_policy", "default"),
```

Comment:
can you please add a dedicated test for this attribute?

Reply:
added TestAccFSxOntapVolume_snapshotPolicy

```
❯ make testacc TESTS=TestAccFSxOntapVolume_snapshotPolicy PKG=fsx
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/fsx/... -v -count 1 -parallel 20 -run='TestAccFSxOntapVolume_snapshotPolicy'  -timeout 180m
=== RUN   TestAccFSxOntapVolume_snapshotPolicy
=== PAUSE TestAccFSxOntapVolume_snapshotPolicy
=== CONT  TestAccFSxOntapVolume_snapshotPolicy
--- PASS: TestAccFSxOntapVolume_snapshotPolicy (2566.39s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/fsx        2569.485s
```